src/main/java/com/example/demo/controller/StudentController.java

```diff
@@ -21,16 +24,16 @@
 @RequestMapping(path = "api/v1/student")
 public class StudentController {
 
-    private final StudentService studentService;
+    private final StudentServiceImpl studentService;
 
     @Autowired
-    public StudentController(StudentService studentService) {
+    public StudentController(StudentServiceImpl studentService) {
         this.studentService = studentService;
     }
 
     @GetMapping
-    public List<Student> getStudent() {
-        return studentService.getStudents();
+    public ResponseEntity<List<Student>> getStudent() {
```

Comment:
I believe I do not need `ResponseEntity`. Let's search and delete if not necessary

Reply:
Tested and assured that it doesn't make any change because if the process is not interrupted, the return is 200 OK. Therefore, changes have been abandoned